.circleci/config.yml

```diff
@@ -83,26 +82,27 @@ executors:
SEZhOCtib0hsQXdKUjYyRWwwM0Ruam1KVXZPQ2lsU1NBWWlJQTZQcGZsSA0KZEFRYVA5bGJJcVRxUDZRRmZJa0pnT0JhbzV0bUN2RVB3N3luN1pmVEdUVG5sM3JFRFZmZWIvYXJYMHVyYU5VWA0KUEZHcm9LSThkZEY1Ti9uQlFaU0pLL0pkUkMxUDBaUlM4WGlLMGNPYjh0L2F5UGF2alRlT1NWQUVKMnI3UGlKMA0KMWJ6b21MZWtDTWdrSkxhU2VQQzJkYjRVQ1NVNkhzcnRPSVFaVEJKNVcwREI4a3NZamlUWUhBaWlUaVI4aXdjbg0KeXRpMTlVYmg4OEVsajZSRHA5b0RjbHJCN21QV0JoNG9vRVI1NVVzVENPd2NZQ2VVVW1zaHNzNGV3czJ6d0crcw0KTVU5R2t3aEphWk1KQkpQdWFJNThIOFgxZHc4eDI3emlZU3BaUGtFQkxLNXVCQzk3SjZMK05qUFRjZWdWaFFocw0KVmw4OTBIQ2NuN3VJM1ZaR3IwMHBIbklDRlJhV01qaUFURURYZVNXZGZwL2lQeVlBclFLQ0FRRUE3b1hORUpaYg0KdUIvbHJpQjhOVHQ1MG0zSExwVHg1VXVnd2JGUnIzSll1L1V1VmFyUnRxRW1pWEkza05GeHMxY3pSc3NVM085Sw0KQS9wT0Y1S3BUaTU3Y3V3MVZ6UnhTZDRKR3EzbHVkeWZGUnFGYzNiRGlTMnp4TFhTK3NFRjlhUnhsbWJFMVgvVA0KZWFZNnZzandYelMycUxGOUtjSjFuM05BcHJHV01mc2o4TTdabG9qTzBaanRtZWlSN3JpTTdLQmpVTHZsdFJ1bw0KbVhvV2FXWWF5VWR2dWlxbG5CdGZIUFZSVzdRVmJuSlBDR3RJTGxtYW1lZkRJOEVscnhNMHp3cFpqNFhEK0h2Rw0KdTRoNXR5dG5qQjJNL05RTzBONC9VMUx5YnVKV2lsc1Q3dnM3SHZRS3gwTTFqenZpNVdNNlh6UzRZTnd0MmlrTA0KWWFzeEt1NnBZOXFvcHdLQ0FRRUE2UzNmS3dqSVQ4MWg3eCtEeG82OUxjd1VaeWRDekwwbXN1SVFqL3h1T0tGSw0KU1lieEswMms5M2YvMGNlRVV2dzBpUjZKOVBSYzVNRlcxZFBHdml1cVc1NW5kcHFpU2N0dWJneDVPQWQrM2FqZA0KeDlwM0FJV3cveVplMDZBMkljSjJvT1BaSHdsdjVncExnekdFZlQzQnRUOExFYjNWZTV5QW1PZFZaWDVabDVUaA0KRkhTQytHNzZWV0xjVHVMc1FlY0xVOFhqcDNwSmxPTUhWZ3oyOVg3QTlGdTQ0VnpjbWVsVS9WbTFFd1dNQ2tQaw0KUWVlSERDRnBhOFVJL2hPSjdrQklQYUdKNDBQTlFTU0VudVB1bSs3THZHSThPNGtTbFFaQjFWM3BZN3FBV3FjNA0Kb01jOGViOWpOVjd3UHpjdGhuNmlPQk11UDFsTXlyejk4Y2luUjI0alN3S0NBUUJFNzV6RngzNzRweGRkalB2NA0KOHBpUnpiSjhid1crYkc0b3JOUGI3aU5oSjBwZDdnR3dVOERzaG5zeXVxV3UzR3Z3Y084MXIxN242V2E2ZjJPNQ0KT2N2cCtnU3NrSGFSajc5MzIzeUNqQlhhSnBFZFB0emlmN2Z1Q1ZMUnNBbkY1ZUlDUXU5dWVRUW9xM2RjL253aQ0KTi96Mm9sMTNQVUY3ZFBiaGRuL2ZNQ05DRExac1FGdnR6ZElWNGxhUzlQbWR4ZmtPTTAvL1dPQVFnMjJJeHI3Rg0KSDRWVis4OWpYN0hNajhZVXgrWk54bkc3RUM2TFh0QkUzT3ZVSUpXQWxQOTFPeHVKVkhMQUZSM3B3MWJTTHBROQ0KdGFpeXhhTUdrTVVpa3QySVdKZ0VZckJQS0lMOHZqR0RSNVJxTWpld2JrNUhoOWdsL2x5dUthQ2xlTEc4SThBSA0KSVFJUEFvSUJBRzVQd1MvK3E2VDFLTXVUUVNnQ0J3VlBjcWVLTDFQa2VCWlFWRDJKMGlzQVFzQ2VFaUhHVGJDMQ0KR0ZVSjZsUEg5S0taeS9JU1dJL3YxMEM1TzJCa0QrZmYvRnBqdFRyci9vWVphSTlWSEhPSW04TWNXd2ZvTjErOQ0KcTBhNm1HRklPZzdMTzhBdkdyemRTZlA2elE3aTFHMEdNaHBta0VRMk9iQWNZSTN2UkRzdHQzMlIwUlB1akljdw0KaWNEWUlhVGJ4TVFRN2k0ZzhBWjdrTkRvdVR4U2hyNFRWbGRuTXdXdEdRUHlHSE96UUJqckhXVUFQeWRCOXptQQ0Kc1V4aHVmTDRmalNKNDAyY21kR2prUW9LRXF1NlFMQVNXU1JoS0dWUDNIMHdQR1Z0cWFNNVVOVGU3SUU4UlpiTw0KSHlJSTNJU3dDOVl6WE1HallXellFOXI2YmdpZzFoa0NnZ0VBY09WbnpBRlJNQWhyc3JBTjVjTmdabS9VWUthVA0KVXozeFBKNStLVXp2b0w4a2t0VHgyVTVRempkenVlZmxiOVVDT1AveWp6TElnUlRhTnRKdjZVODRURUNWSVQyRA0KVUtIVkdlQ0hGUE5MZU9UbE5PTVVjOFo0SGw1Tzd4MjcwQXhXTTJndXBncUVNUGIxbnJnWGF2ekNDZ2lSU2lnZQ0KbUx6a0FpOElEYzduZkxPUXBjaGVNSG02TkNBcDRRZ1pFYkRaYmp6VUdrUTlRMGNlT0wwLy9JYzIwdmQ1a2pWbw0KTjI5dnJuYStCTW0rejJ3Q2FJSFZmNWlWbzhFdGxwN2E2OEJnQkpEUVVxTzh4M1hpMWhCWkxuWlNJSEFWYjZXLw0KaGd1UmVkRlJucXQ5QWhVdVN0ZlRzUExoUDlXNGN5bDd2QzcyZ3k4K0lkbTgwTkFCaGpaUE0zelF1QT09DQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQ==
```

Comment:
The `JWT_SIGNING_KEY` env vars are pretty lengthy, and we repeat them multiple times in the CircleCI config. Is there a way to only define them once and then re-use them in the different environments that need them?

Reply:
Actually I think we can take them out completely- they’re available as env vars from the org context.